lib/ChainwebData/Genesis.hs

```diff
@@ -0,0 +1,13 @@
+{-# LANGUAGE NumericUnderscores #-}
+module ChainwebData.Genesis
+( genesisHeight
+) where
+
+import Chainweb.Api.ChainId (ChainId(..))
+
+
+genesisHeight :: ChainId -> Int
+genesisHeight (ChainId c)
+  | c `elem` [0..9] = 0
+  | c `elem` [10..19] = 852_054
```

Comment:
I don't think we should hard code these.  In the future we will probably start running chainweb-data for other networks like testnet, so we should get this information from the node.  It is available from the `/info` endpoint which is already being queried and made available in the environment.

Reply:
This now uses the environment `NodeInfo` object to determine lower bounds for genesis.